ddtrace/propagation/http.py

```diff
@@ -120,24 +120,24 @@ def _inject(span_context, headers):
         if span_context.dd_origin is not None:
             headers[HTTP_HEADER_ORIGIN] = ensure_text(span_context.dd_origin)
 
+        # Skip injecting tracer tags
+        if not config._propagation_datadog_tags_enabled:
```

Comment:
We need to add a meta tag to the trace saying we didn't propagate because it is disabled.

Reply:
should we only do this if there are tags and propagation is disabled? seems unnecessary to add it when there are no tags received?